web/src/lightbox.ts

```diff
@@ -619,6 +621,23 @@ function remove_video_players(): void {
     $("#lightbox_overlay .video-player").html("");
 }
 
+export function handle_inline_media_element_click(
+    e: JQuery.ClickEvent,
+    $media: JQuery<HTMLMediaElement> | JQuery<HTMLImageElement>,
+): void {
+    // prevent the link from opening in a new page.
+    e.preventDefault();
+    // prevent the message compose dialog from happening.
+    e.stopPropagation();
```

Comment:
Let's keep this in the event handler call itself to increase the reusability of this function, readability of the event handler and avoid unexpected behaviour with future changes.